src/components/common/Alpha.js

```diff
@@ -5,8 +5,24 @@ import * as alpha from '../../helpers/alpha'
 import Checkboard from './Checkboard'
 
 export class Alpha extends (PureComponent || Component) {
+  constructor(props) {
+    super(props)
+
+    this.container = null
+  }
+  
+  componentDidMount() {
+    // e.preventDefault() does not work properly on React synthetic touch event.
+    // As a workaround, touch events have to be added to directly to node
+    // https://github.com/facebook/react/issues/9809#issuecomment-414072263
```

Comment:
follow up here, the comment you're referencing doesn't actually show the solution.

You need to update your use to one of these:
- add a `{ passive: false }` option to the `addEventListener`
  - based on this solution https://github.com/facebook/react/issues/9809#issuecomment-407690524
- use `el.ontouchstart`
  - based on this solution https://github.com/facebook/react/issues/9809#issuecomment-413978405

Note if you use the `{ passive: false }` option route, you should probably do the check to see if it is supported too.

Reply:
@hartzis I have used the second option. I used `el.addEventListener('touchstart')` instead of `el.ontouchstart`. Aren't those same? Please correct me if I'm wrong.

Reply:
They are different. Pretty helpful explanation: https://stackoverflow.com/a/35093997

In order to gain the ability to call `preventDefault` you will need to add also add the `{ passive: false }` option for chrome.
```
this.container.addEventListener('touchstart', this.handleChange, {passive: false})
```
**or** do what dan recomended
```
this.container.ontouchstart = this.handleChange
```

Let me know if this helps, i just stumbled across this PR since it referenced a facebook issue i was looking at and i'm dealing with a lot of the same issues as the maintainer for https://github.com/dogfessional/react-swipeable

Reply:
@hartzis Thanks for the information. After looking into this issue more, I found that `ontouchstart` is currently an experimental feature. Also in Chrome, the flag `touch-event` needs to be enabled to use `ontouchstart` API.  https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/ontouchstart
I'm not sure why dan used `ontouchstart`...

Regarding `passive` option, I think it does not need to be set to `false` unless the event handler is attached to document level e.g. `window`, `document` or `body`.  [Document level event handlers are treated as passive](https://www.chromestatus.com/features/5093566007214080)

Reply:
@cokencode That is an incredibly helpful link, thank you for sharing. I either totally missed or forgot the `window, document or body` part of that change. You rock!